backend/common/src/main/java/ai/verta/modeldb/common/CommonDBUtil.java

```diff
@@ -318,6 +318,11 @@ protected static void createDBIfNotExists(RdbConfig rdbConfiguration) throws SQL
     Connection connection =
         DriverManager.getConnection(
             dbUrl, rdbConfiguration.getRdbUsername(), rdbConfiguration.getRdbPassword());
+
+    if (rdbConfiguration.isH2()) {
+      LOGGER.debug("database driver is H2...skipping creation");
+      return;
+    }
```

Comment:
Do we not need to create the DB when using H2?

Reply:
H2 auto-creates the DB when it's accessed.